dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -0,0 +1,95 @@
+:css
+  body {
+    font-family: Gotham, sans-serif;
+  }
+
+%p
+  Hi
+  = @teacher.name.split().first.titleize + ','
+
+%p
+  Thank you for attending a Code.org
+  = @workshop.course
+  = @workshop.subject
+  workshop and for your commitment to bring foundational 21st century learning to your students!
+
+- if @is_first_workshop
+  %p
+    Computer science has become the fastest-growing subject in K-12 education,
+    thanks to inspirational educators like you who have decided to spend your summer or weekends
+    in professional learning for the benefit of your classroom and the future of our children.
+    Thank you for joining the teacher-powered movement to give every student in every school
+    the opportunity to learn computer science.
+
+%p
+  = link_to('Please take a minute to complete this short survey', @survey_url)
+  Let us know how we can improve your experience and your learning.
+  If you had a good experience, please spread the word about the
+  = link_to('Code.org Professional Learning Program', 'https://code.org/educate') + '.'
+
+= render partial: 'share_links'
+
+- if @workshop.course == Pd::Workshop::COURSE_CSF
+  %h3
+    Classroom supplies, at no cost:
+  %p
+    Completing the survey will qualify you to receive supplies at no cost for the unplugged activities.
+
+  %p
+    Attached to this email, you will also find a personalized certificate acknowledging your successful
+    completion of Code.org's CS Fundamentals workshop.
+
+  %h3
+    For a limited time
+  %p
+    Code.org is offering $50 off
+    = link_to('Dash robots', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/Mzc1ODY2') + '.'
```

Comment:
Are these links referral links? I think in the email it should be the direct URL and each email generates its own referral link?